### DISCUSSION TITLE
[video] Typing in Engram on a split columnar keyboard

### DISCUSSION THREAD

**Discussion by sunaku** [binarybottle/engram-layout#70] [Show and tell]
Hey Arno and Engram community,

I've recorded [a basic video demonstration :movie_camera: of me typing](https://vimeo.com/873206293) in Arno's Engram 2.0 layout with [Miryoku home row mods *enabled*](https://sunaku.github.io/home-row-mods.html) :fire: of [my Glove80 keyboard](https://sunaku.github.io/moergo-glove80-keyboard.html) with Kailh Choc V1 Red switches.

>Video: (click to play) :play_or_pause_button:
>[![video](https://i.vimeocdn.com/video/1742895020-f55326b765cd6fbfc224a558086fac7032e6fd3798f568b8eb120da6824534b5-d_640)](https://vimeo.com/873206293)

A few things to notice in the video: :tipping_hand_man:
- My 8 touch-typing fingers stay in their lanes :swimming_man: (no reaching into the central columns).
- The speed and rhythm of the text cursor as it travels over the words on the screen; you can trace this back to inward rolls in Engram layout --- for example, consider the word "you".
- I stumbled towards the end and lost my flow :sweat_smile: so it's not my best performance, but okay.

Cheers.

**Comment by binarybottle** [comment]
Thank you for the demonstration and comments, @sunaku!  It's wonderful to see your typing speed and rhythm with the layout!

**Comment by sunaku** [comment]
Here is another video: typing Rust code :crab: in Engram using [my Symbol Layer](https://sunaku.github.io/moergo-glove80-keyboard.html#symbol-layer). :symbols:

>Video: (click to play) :play_or_pause_button:
>[![video](https://i.ytimg.com/vi/fPcO5NGoDao/hqdefault.jpg)](https://youtu.be/fPcO5NGoDao)
